qadence_protocols/mitigations/readout.py

```diff
@@ -14,8 +13,32 @@
 from scipy.optimize import LinearConstraint, minimize
 
 
-def corrected_probas(p_corr: npt.NDArray, T: npt.NDArray, p_raw: npt.NDArray) -> np.double:
-    return norm(T @ p_corr.T - p_raw.T, ord=2) ** 2
+def tensor_rank_mult(qubit_ops: torch.Tensor, prob_vect: npt.NDArray) -> npt.NDArray:
```

Comment:
Here and everywhere else that you @rajaiitp use numpy instead of torch is it for faster computations, or some functionalities missing in torch?

Reply:
@gvelikova : I did discuss this with @Roland-djee the other day, we decided to leave it as numpy unless any torch related functionalities (like gradients) are seen necessary. Let me know if you think otherwise.